tests/test_query.py

```diff
@@ -172,18 +172,20 @@ def test_get_invalid_age(
     assert response.status_code == 422
 
 
-@pytest.mark.parametrize("valid_sex", ["male", "female", "other"])
+@pytest.mark.parametrize(
+    "valid_sex", ["248153007", "248152002", "32570681000036106"]
```

Comment:
I think the parameters should take the form `snomed:some_id` since the `CONTROLLED_TERM_REGEX` is checking for that.

Reply:
Do you mean you'd prefer to have the `snomed` prefix in front of each of the parameters instead of appended in the query URL path?
```
   response = test_app.get(f"/query/?sex=snomed:{valid_sex}")
```